README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
+
+## The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; and if you can’t resolve an issue contact the admins. If you do not follow our Community Code of Conduct, it will be apparent and you may be removed from HangOps.
```

Comment:
:thumbsup: Yep we probably need to find a better word than "bothered".

I'd like to be careful about treating all possible disrespectful interactions as victim/perpetrator situations and instead lean on the history of the overwhelmingly positive interactions we've had in HangOps. I think the responsibility has to start with us as individuals if we see something uncool happening we need to speak up. That's why I find "respectfully ask people to stop" powerful. It puts the ownership on us as individuals and leaves this CoC as a process of last resort.